pkgs/development/python-modules/imagededup/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cython
+, torch
+, torchvision
+, pillow
+, tqdm
+, scikit-learn
+, pywavelets
+, matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "imagededup";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "idealo";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-B2IuNMTZnzBi6IxrHBoMDsmIcqGQpznd/2f1XKo1Oa4=";
+  };
+
+  propagatedBuildInputs = [
+    cython
+    torch
+    torchvision
+    pillow
+    tqdm
+    scikit-learn
+    pywavelets
+    matplotlib
+  ];
+
+  doCheck = false;
```

Comment:
Why don't you run the tests?
If you want to disable it, you need to leave a comment.

Reply:
is it fine this way?

Reply:
How about using `export HOME=$TMPDIR` in preCheck?

Reply:
tests are now executed. I am not sure if the preloading of the models it fine?